### ISSUE TITLE
Reduce duplication of versions and streamline release process

### ISSUE THREAD

**Issue by berezovskyi** [OSLC/oslc4net#75]
I would like to:

1. Have common properties shared across a subset of projects within the solution. Right now, every project includes a set of properties show below. Primarily, I would like to update `VersionSuffix` and `Version` in a single place.
2. Synchronize the updates to the MSBuild `Version`, assembly `AssemblyVersion` as well as `AssemblyFileVersion` (which [CLR doesn't care about](https://stackoverflow.com/a/65062/464590), apparently, but which we should still keep in sync unless we can completely drop it).

```xml
<Version>0.3.0-alpha</Version>
<VersionSuffix>alpha</VersionSuffix>
<PackageLicenseExpression>EPL-1.0</PackageLicenseExpression>
<Authors>Andrii Berezovskyi and OSLC4Net contributors</Authors>
<PackageReadmeFile>README.md</PackageReadmeFile>
<!-- TODO: remove PackageOutputPath after migration to .NET Core-->
<PackageOutputPath>MSBuild_Nuget_Pack/</PackageOutputPath> 
<PackageProjectUrl>https://github.com/OSLC/oslc4net/</PackageProjectUrl>  <RepositoryUrl>https://github.com/OSLC/oslc4net/</RepositoryUrl> 
```

**Comment by berezovskyi** [comment]
The process was started by introducing https://github.com/OSLC/oslc4net/blob/master/OSLC4Net_SDK/Directory.Build.props

The AssemblyVersion issue went away as we migrated to net6.0+.

**Comment by berezovskyi** [comment]
fixed in #113
